### PR TITLE
feat(rule): Enable arrow-body-style

### PR DESCRIPTION
**Pull request by EvanPurkhiser**
See https://eslint.org/docs/rules/arrow-body-style

### CODE REVIEW

File: packages/eslint-config-sentry/rules/base.js [hunk 1]
```diff
@@ -256,6 +256,9 @@ module.exports = {
     // http://eslint.org/docs/rules/wrap-iife
     'wrap-iife': ['error', 'any'],
 
+    // https://eslint.org/docs/rules/arrow-body-style
+    "arrow-body-style": ["error", "always"],
```
Comment (EvanPurkhiser) [inline packages/eslint-config-sentry/rules/base.js]: oops, not always, that's wrong. fixing

**Comment by billyvg** [comment]
Anyone in @getsentry/app-frontend have thoughts about this?

**Comment by EvanPurkhiser** [comment]
I'll have a diff in a second for sentry.

**Comment by priscilawebdev** [comment]
I like the `"as-needed" `option 👍

**Comment by priscilawebdev** [comment]
@markstory do you know the which prettier rule is that? I could not find anything.. there is the https://prettier.io/docs/en/options.html#arrow-function-parentheses but it is something else

**Comment by EvanPurkhiser** [comment]
> @markstory do you know the which prettier rule is that? I could not find anything.. there is the https://prettier.io/docs/en/options.html#arrow-function-parentheses but it is something else

AFAIK There is no prettier rule to enforce this.

**Pull request merged**